Support/Scripts/run-lldb-tests.sh

```diff
@@ -13,8 +13,34 @@
 # against ds2. It requires a few hacks in the testing infra to disable
 # broken unit tests.
 
+cherry_pick_patches() {
+  if ! $opt_fast; then
+    cd "$1"
+
+    testPath="$top/Support/Testing"
+
+    # Disabled tests
+    for p in $testPath/Patches/*.patch ; do
+      echo "Applying $p"
+      patch -d "$lldb_path" -p1 <"$p"
+    done
+
+    # Upstream patches
+    while read line; do
+      hash=$(echo $line | awk '{print $1}')
+      echo "Cherry-picking upstream commit: $hash"
+      git cherry-pick --no-commit "$hash"
+    done < $testPath/Patches/Upstream-release_38-patches.txt
+
+    rm -fr "$lldb_path/packages/Python/lldbsuite/test/functionalities/watchpoint"
```

Comment:
You should move that under `# Disabled tests` and add a small comment explaining why we're doing this.